openslides_backend/action/util/assert_belongs_to_meeting.py

```diff
@@ -15,6 +15,10 @@ def assert_belongs_to_meeting(
 
     errors: List[FullQualifiedId] = []
     for fqid in fqids:
+        if fqid.collection.collection == "meeting":
+            if fqid.id == meeting_id:
+                continue
+            errors.append(fqid)
```

Comment:
A `continue` or `else` is missing here, for meetings we do not need to fetch anything from the datastore.

Reply:
done